eng/pipelines/docindex.yml

```diff
@@ -16,6 +15,8 @@ jobs:
       - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
         parameters:
           SkipCheckoutNone: true
+          Paths: 
+            - .github/
```

Comment:
Why is the .github folder needed?

Reply:
It is needed for service code owner file. https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS

We can limit to CODEOWNER file as well.

Reply:
I'd suggest either being explicit or at least adding a comment as to why this is needed otherwise someone is likely to remove it.

Reply:
Done